packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -155,5 +155,13 @@ export const messages = {
   error_creating_packagexml: 'Error creating package.xml. %s',
   error_extracting_org_source: 'Error extracting downloaded Apex source. %s',
   error_extracting_packages: 'Error extracting packages: %s',
-  error_updating_sfdx_project: 'Error updating sfdx-project.json: %s'
+  error_updating_sfdx_project: 'Error updating sfdx-project.json: %s',
+
+  demo_mode_status_text: `$(gist-secret) SFDX DEMO`,
+  demo_mode_status_tooltip:
+    'You are running the SFDX VS Code extensions in demo mode. You will be prompted if you are connecting to production orgs.',
+  demo_mode_prompt:
+    'Authenticating against a business or production org is not recommended on a demo or shared machine. Continue authenticating or cancel?',
+
+  force_auth_logout_all_text: 'SFDX: Log Out from All Authorized Orgs'
```

Comment:
![screen shot 2018-03-06 at 3 32 49 pm](https://user-images.githubusercontent.com/54414/37065398-828edb44-2155-11e8-8e29-bd9254d7ca48.png)

![untitled](https://user-images.githubusercontent.com/54414/37065475-e0e601c2-2155-11e8-9719-a83ae7d78be7.png)


CC: @ruthemmanuelle

Reply:
By the way, this style of prompt is in VS Code Insiders. It's not in the normal release yet but will come out soon in early March. Just wanted to use this since it is what will be available to users soon.

Reply:
@ruthemmanuelle - Updated text.

![screen shot 2018-03-06 at 4 10 00 pm](https://user-images.githubusercontent.com/54414/37066156-dfa8dff2-2158-11e8-8cb1-11706abf855c.png)